communities/microgalaxy/lab/sections/1_data_import_and_preparation.yml

```diff
@@ -60,6 +71,31 @@ tabs:
               - fastq
               - fasta
         button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fkraken2%2Fkraken2"
```

Comment:
I would add:
- dada2
- (meta)spades
- unicycler

Reply:
only dada2 i wont be able to add since dada2 is a collection of tools which one shall I choose?

Reply:
You could do as in the tool section.